doc/nrf/releases/release-notes-2.2.0.rst

```diff
@@ -0,0 +1,935 @@
+  * An issue with passing quoted settings to child images.
+
+Wi-Fi
+-----
+
+See `Wi-Fi samples`_ for details about how to use Wi-Fi in your application.
+
+Protocols
+=========
+
+This section provides detailed lists of changes by :ref:`protocol <protocols>`.
+See `Samples`_ for lists of changes for the protocol-related samples.
+
+Bluetooth LE
+------------
+
+* Added:
+
+  * Support for :c:func:`hci_driver_close`, so :c:func:`bt_disable` can now be used to disable the :ref:`softdevice_controller`.
+  * The Kconfig option :kconfig:option:`CONFIG_BT_UNINIT_MPSL_ON_DISABLE` that, when enabled, uninitializes the MPSL when :c:func:`bt_disable` is used.
+    This releases all peripherals used by the MPSL.
+
+For details, see the :ref:`SoftDevice Controller changelog <nrfxlib:softdevice_controller_changelog>`.
+
+Bluetooth mesh
+--------------
+
+* Added:
+
+  * Documentation pages:
+
+    * :ref:`ug_bt_mesh_fota`.
+    * :ref:`ug_bt_mesh_node_removal`.
+
+Also, see `Bluetooth mesh samples`_ for the list of changes.
+
+Enhanced ShockBurst (ESB)
+-------------------------
+
+* Added the :kconfig:option:`CONFIG_ESB_DYNAMIC_INTERRUPTS` Kconfig option to enable direct dynamic interrupts.
+
+nRF IEEE 802.15.4 radio driver
+------------------------------
+
+* Added the functionality where Radio trim values are reapplied after a ``POWER`` register write as a workaround for the hardware errata 158 of the nRF5340 chip.
+
+Matter
+------
+
+* Added:
+
+  * Documentation pages:
+
+    * :ref:`ug_matter_overview_dfu`.
+    * :ref:`ug_matter_overview_multi_fabrics` and entry about binding to :ref:`ug_matter_network_topologies_concepts`.
+    * :ref:`ug_matter_overview_commissioning`, which is based on an earlier subsection of :ref:`ug_matter_overview_network_topologies`.
+    * :ref:`ug_matter_overview_security`, which is based on an earlier subsection of :ref:`ug_matter_overview_network_topologies`.
+    * :ref:`ug_matter_device_certification_reqs_security` on the page about :ref:`ug_matter_device_certification`.
+
+* Updated:
+
+  * :ref:`ug_matter_overview_int_model` with an example of the interaction.
+  * :ref:`ug_matter_overview_data_model` with an example of the Data Model of a door lock device.
+  * :ref:`ug_matter_gs_adding_cluster` documentation with new code snippets to align it with the source code of refactored Matter template sample.
+
+See `Matter samples`_ for the list of changes for the Matter samples.
+
+Matter fork
++++++++++++
+
+The Matter fork in the |NCS| (``sdk-connectedhomeip``) contains all commits from the upstream Matter repository up to, and including, ``bc6b43882a56ddb3e94d3e64956bd5f3292b4058``.
+
+The following list summarizes the most important changes inherited from the upstream Matter:
+
+* |no_changes_yet_note|
+
+Thread
+------
+
+* Added experimental TCP support as required by Thread 1.3 Specification.
+
+See `Thread samples`_ for the list of changes for the Thread samples.
+
+HomeKit
```

Comment:
As confirmed with @mariusz-nordicsemi , this section should be removed as there are no items to be listed for this release

Reply:
thanks for checking!